byond-extools/src/monstermos/atmos_defines.h

```diff
@@ -1,133 +1,139 @@
 #pragma once
 
+namespace monstermos::constants
+{
+
 //ATMOS
 //stuff you should probably leave well alone!
-#define R_IDEAL_GAS_EQUATION	8.31	//kPa*L/(K*mol)
-#define ONE_ATMOSPHERE			101.325	//kPa
-#define TCMB					2.7		// -270.3degC
-#define TCRYO					225		// -48.15degC
-#define T0C						273.15	// 0degC
-#define T20C					293.15	// 20degC
-
-#define MOLES_CELLSTANDARD		(ONE_ATMOSPHERE*CELL_VOLUME/(T20C*R_IDEAL_GAS_EQUATION))	//moles in a 2.5 m^3 cell at 101.325 Pa and 20 degC
-#define M_CELL_WITH_RATIO		(MOLES_CELLSTANDARD * 0.005) //compared against for superconductivity
-#define O2STANDARD				0.21	//percentage of oxygen in a normal mixture of air
-#define N2STANDARD				0.79	//same but for nitrogen
-#define MOLES_O2STANDARD		(MOLES_CELLSTANDARD*O2STANDARD)	// O2 standard value (21%)
-#define MOLES_N2STANDARD		(MOLES_CELLSTANDARD*N2STANDARD)	// N2 standard value (79%)
-#define CELL_VOLUME				2500.0	//liters in a cell
-#define BREATH_VOLUME			0.5		//liters in a normal breath
-#define BREATH_PERCENTAGE		(BREATH_VOLUME/CELL_VOLUME)					//Amount of air to take a from a tile
+const float R_IDEAL_GAS_EQUATION = 8.31;	//kPa*L/(K*mol)
+const float ONE_ATMOSPHERE       = 101.325;	//kPa
+const float TCMB                 = 2.7;		// -270.3degC
+const float TCRYO                = 225;		// -48.15degC
+const float T0C	                 = 273.15;	// 0degC
+const float T20C                 = 293.15;	// 20degC
+
+const float CELL_VOLUME				= 2500.0;	//liters in a cell
+const float MOLES_CELLSTANDARD		= (ONE_ATMOSPHERE*CELL_VOLUME/(T20C*R_IDEAL_GAS_EQUATION));	//moles in a 2.5 m^3 cell at 101.325 Pa and 20 degC

```

Comment:
Funny story. This was originally calculated without `CELL_VOLUME` having been defined.